sql/moz-fx-data-shared-prod/mdn_yari_derived/mdn_popularities_v1/query.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
+
+"""Monthly data exports of MDN 'Popularities'. This aggregates and counts total page visits and normalizes them agains the max."""
+import logging
+from argparse import ArgumentParser
+from datetime import datetime
+
+from google.cloud import bigquery, storage
+
+QUERY_TEMPLATE = """\
```

Comment:
Have you tested this? I think this query might fail because:
1. It's not filtering over `submission_date`
2. `@submission_date` parameter is not passed in (see https://github.com/mozilla/bigquery-etl/blob/7b00dd676df06147009050e27d2ea45eba996abf/script/glam/run_scalar_agg_clustered_query.py#L70)

Reply:
Thanks, I just noticed I was using both @submission_date and date placeholder. Updated to use the suggested approach.